tests/csm/rest/test_capacity_usage.py

```diff
@@ -1084,3 +1084,423 @@ def test_39923(self):
             break
         self.log.info("[END] completed")
         self.deploy = True
+
+class TestSystemCapacityFixedPlacement():
+    """System Capacity Testsuite"""
+
+    # pylint: disable-msg=too-many-statements
+    @classmethod
+    def setup_class(cls):
+        """ This is method is for test suite set-up """
+        cls.log = logging.getLogger(__name__)
+        cls.log.info("[START] Setup Class")
+        cls.csm_obj = csm_api_factory("rest")
+        cls.log.info("Initiating Rest Client ...")
+        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.username = cls.csm_obj.config["csm_admin_user"]["username"]
+        cls.user_pass = cls.csm_obj.config["csm_admin_user"]["password"]
+        cls.deploy = False
+        cls.akey = ""
+        cls.skey = ""
+        cls.s3_user = ""
+        cls.bucket = ""
+        cls.node_list = []
+        cls.num_nodes = len(CMN_CFG["nodes"])
+        cls.io_bucket_name = f"iobkt1-copyobject-{time.perf_counter_ns()}"
+        cls.s3_obj = s3_test_lib.S3TestLib()
+        cls.ha_obj = HAK8s()
+        for node in CMN_CFG["nodes"]:
+            cls.log.debug("Worker node : %s", node["hostname"])
+            cls.node_list.append(LogicalNode(hostname=node["hostname"],
+                                                 username=node["username"],
```

Comment:
Can u run autopep for correcting all PEP8 mistakes?

Reply:
committed autopep8 changes for entire file